### DISCUSSION TITLE
Cant showing DV/V plot

### DISCUSSION THREAD

**Discussion by EvanFadhilahR** [ROBelgium/MSNoise#333] [Q&A]
Dear Thomas.
may i ask for solution, i just tried to plot dv/v but it cant show the graphic on the picture, is there any mistake on commend or something? thanks !!
![how](https://github.com/ROBelgium/MSNoise/assets/163017878/b336998b-6c91-4245-833b-513cfba9dfa8)

**Comment by EvanFadhilahR** [comment]
i just check on msnoise admin the dv/v just look like that
![tanya](https://github.com/ROBelgium/MSNoise/assets/163017878/7c8795fd-9a88-432a-b320-45b263f413da)

**Comment by ThomasLecocq** [comment]
the -p "pair" option is not yet implemented. if all steps worked properly (you might want to run the msnoise commands with "-v" to see debug messages too) , the DVV2 folder should contain files and those should result on data plotted.

**Reply by EvanFadhilahR** [reply]
![tanyaaa](https://github.com/ROBelgium/MSNoise/assets/163017878/1eb5659a-3efe-47cc-a287-37fd02e4cca8)
i just found folder that contain all.nc and zz.nc but still can't plot dvv, is there any posiblelities mistakes?

**Reply by ThomasLecocq** [reply]
Yes, the problem is that you're mixing two versions of the development code, you should restart a fresh project and pay attention to the mov_stack parameter.

**Reply by EvanFadhilahR** [reply]
I still dont really understand about mov_stack paramer if i tried to running 60 days record and set step windows avg 2d, 3d. 5d, 10 day is it true to input like this sir ?
![image](https://github.com/ROBelgium/MSNoise/assets/163017878/77b0a3d9-6319-422c-a04c-cf509a183ad5)
